Client/Frontend/Home/PrivateHome/PrivateHomepageViewController.swift

```diff
@@ -0,0 +1,132 @@
+         notificationCenter: NotificationProtocol = NotificationCenter.default
+    ) {
+        self.themeManager = themeManager
+        self.notificationCenter = notificationCenter
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupLayout()
+
+        listenForThemeChange(view)
+        applyTheme()
+    }
+
+    private func setupLayout() {
+        scrollContainer.addArrangedSubview(logoHeaderCell.contentView)
+        scrollContainer.addArrangedSubview(privateMessageCardCell)
+        scrollContainer.accessibilityElements = [logoHeaderCell.contentView, privateMessageCardCell]
+
+        view.layer.addSublayer(gradient)
+        view.addSubview(scrollView)
+        scrollView.addSubview(scrollContainer)
+
+        let contentLayoutGuide = scrollView.contentLayoutGuide
+        let frameLayoutGuide = scrollView.frameLayoutGuide
+
+        NSLayoutConstraint.activate([
+            scrollView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+
+            scrollContainer.topAnchor.constraint(equalTo: contentLayoutGuide.topAnchor,
+                                                 constant: UX.defaultScrollContainerPadding),
+            scrollContainer.bottomAnchor.constraint(equalTo: contentLayoutGuide.bottomAnchor,
+                                                    constant: -UX.defaultScrollContainerPadding),
+            scrollContainer.leadingAnchor.constraint(equalTo: contentLayoutGuide.leadingAnchor,
+                                                     constant: UX.scrollContainerPadding),
+            scrollContainer.trailingAnchor.constraint(equalTo: contentLayoutGuide.trailingAnchor,
+                                                      constant: -UX.scrollContainerPadding),
+
+            scrollContainer.widthAnchor.constraint(equalTo: frameLayoutGuide.widthAnchor,
+                                                   constant: -UX.scrollContainerPadding * 2),
+        ])
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        gradient.frame = view.bounds
+    }
+
+    func applyTheme() {
+        // TODO: Felt Privay - Theming System should be handled by Redux 
+        // https://mozilla-hub.atlassian.net/browse/FXIOS-7879
+        themeManager.changeCurrentTheme(.privateMode)
+        let theme = themeManager.currentTheme
+        gradient.colors = theme.colors.layerHomepage.cgColors
+        logoHeaderCell.applyTheme(theme: theme)
+        privateMessageCardCell.applyTheme(theme: theme)
```

Comment:
this normally will not be needed since your `privateMessageCardCell` is `ThemeApplicable` 🤔 
```suggestion
```

Reply:
ah yes, the updates should be handled by `listenForThemeChange`. thanks for the catch!